python_on_whales/components/context/cli_wrapper.py

```diff
@@ -81,9 +81,40 @@ def use(self) -> None:
 
 
 class ContextCLI(DockerCLICaller):
-    def create(self):
-        """Not yet implemented"""
-        raise NotImplementedError
+    def create(
+            self,
+            context: str,
+            host: str = None,
+            description: str = None,
+            from_: str = None,
+            from_host: ValidContext = None
+    ):
+        """Creates a new context
+
+        # Arguments
+            context: name of the context to create
+            host: uri of the host to access
+            description: description of the context
+            from_: create context from a named context
+            from_host: use endpoint configuration from existing named context
+        """
+        existing_contexts = [str(x) for x in self.list()]
+        if context not in existing_contexts:
```

Comment:
I don't think we should add more logic than what is done in the command line. If the user input will trigger an error, let the error from the command line pop up and it will be sent back to the user as an exception